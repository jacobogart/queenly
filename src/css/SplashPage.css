.SplashPage {
  font-family: "Lobster";
  font-size: 86px;
  border: 1px solid orange;
}

.SplashPage h1 {
  margin: 1rem;
}
