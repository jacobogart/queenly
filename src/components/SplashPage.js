import React from "react";
import SearchBar from "./SearchBar";
import "../css/SplashPage.css";

const SplashPage = (props) => {
  return (
    <section className="SplashPage">
      <h1>Queenly</h1>
      <SearchBar 
        updateResults={props.updateResults}/>
    </section>
  );
};

export default SplashPage;
