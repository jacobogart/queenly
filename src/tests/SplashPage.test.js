import React from 'react';
import SplashPage from '../components/SplashPage';
import { shallow } from 'enzyme';

describe('SpashPage', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <SplashPage />
    )
  });


  it('should render correctly.', () => {
    expect(wrapper).toMatchSnapshot();
  });

});