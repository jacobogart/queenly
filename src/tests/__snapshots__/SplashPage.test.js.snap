// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpashPage should render correctly. 1`] = `
<section
  className="SplashPage"
>
  <h1>
    Queenly
  </h1>
  <SearchBar />
</section>
`;
